src/PCLExt.FileStorage.UWP/StorageFolderImplementation.cs

```diff
@@ -0,0 +1,285 @@
+﻿using PCLExt.FileStorage.UWP.Extensions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.Storage;
+using Windows.Storage.Search;
+
+namespace PCLExt.FileStorage.UWP
+{
+    sealed class StorageFolderImplementation : IFolder
```

Comment:
I do not understand why you are not using the default implementation of both IFolder and IFile. I do not see anything special in UWP

Reply:
Fix async calling in sync code or use the default implementations of IFile and IFolder

Reply:
Default implementation from .Net Standard doesn't allow you access to any files by path https://social.msdn.microsoft.com/Forums/windowsapps/en-US/fdb5260a-db4f-46b6-9332-a7663a351f0d/read-a-file-with-net-standard-20?forum=wpdevelop

Reply:
Well that explains the problem I had back when I tried to do UWP. Do executing via admin grants full permission for file access?

Reply:
It doesn't matter. Initial API for UWP is StorageFolder & StorageFile. File was added in .Net Standard 2.0 only for compatibility. And the File works only for allowed for the app folders. As far as I remember it's only LocalFolder, AppFolder & TempFolder.

Reply:
When I did some research there was a permission for documents folder, but seems it's not working anymore